vitest.config.js

```diff
@@ -0,0 +1,13 @@
+/// <reference types="vitest/config" />
+import { defineConfig } from 'vite';
+
+export default defineConfig({
+    test: {
+        include: ['**/*.test.ts'],
+        workspace: ['packages/*'],
+        coverage: {
+            clean: true,
+            exclude: ['./packages/*/dist', './scripts', './bench', './*.config.js'],
+        },
+    },
+});
```

Comment:
Missing Jest's remapping configs:

https://github.com/inikulin/parse5/blob/2094d5b08d11feda3f164bcf3e1e6aa65a5d0733/package.json#L63-L70

```suggestion
import path from 'node:path';
import { defineConfig } from 'vitest/config';

export default defineConfig({
    test: {
        include: ['**/*.test.ts'],

        // https://github.com/inikulin/parse5/blob/2094d5b08d11feda3f164bcf3e1e6aa65a5d,0733/package.json#L63C1-L70C11
        alias: [
            { find: /^(parse5[^/]*)\/dist\/(.*?)(?:\.js)?$/, replacement: path.resolve('./packages/$1/lib/$2') },
            { find: /^(parse5[^/]*)$/, replacement: path.resolve('./packages/$1/lib/index.ts') },
            { find: /parse5-test-utils\/(.*?)/, replacement: path.resolve('./test/$1') },
        ],

        coverage: {
            include: ['packages'],
            exclude: ['**/dist/**'],
        },
    },
});
```

Reply:
have used your other suggestion from discord 👍